aicsimageio/metadata/utils.py

```diff
@@ -422,3 +425,67 @@ def clean_ome_xml_for_known_issues(xml: str) -> str:
         )
 
     return xml
+
+
+def dtype_to_ome_type(npdtype: np.dtype) -> PixelType:
+    """
+    Convert numpy dtype to OME PixelType
+
+    Parameters
+    ----------
+    npdtype: numpy.dtype
+        A numpy datatype.
+
+    Returns
+    -------
+    ome_type: PixelType
+        One of the supported OME Pixels types
+    """
+    ometypedict = {
+        np.dtype(np.int8): PixelType.INT8,
+        np.dtype(np.int16): PixelType.INT16,
+        np.dtype(np.int32): PixelType.INT32,
+        np.dtype(np.uint8): PixelType.UINT8,
+        np.dtype(np.uint16): PixelType.UINT16,
+        np.dtype(np.uint32): PixelType.UINT32,
+        np.dtype(np.float32): PixelType.FLOAT,
+        np.dtype(np.float64): PixelType.DOUBLE,
+        np.dtype(np.complex64): PixelType.COMPLEXFLOAT,
+        np.dtype(np.complex128): PixelType.COMPLEXDOUBLE,
+    }
+    ptype = ometypedict.get(npdtype)
+    if ptype is None:
+        raise ValueError(f"Ome utils can't resolve pixel type: {npdtype.name}")
+    return ptype
```

Comment:
All the user needs to know is that the data type is unacceptable.  I agree the dicts could be created globally once at init time.  This is just a wrapper around the lookup so that it can be used more concisely.

Reply:
Works for me.